controllers/capabilities/openapi_controller.go

```diff
@@ -220,6 +250,15 @@ func (r *OpenAPIReconciler) reconcileSpec(openapiCR *capabilitiesv1beta1.OpenAPI
 	}
 
 	statusReconciler := NewOpenAPIStatusReconciler(r.BaseReconciler, openapiCR, providerAccount.AdminURLStr, err, productSynced)
+
+	// If the product is successfully synced AND the OpenAPI CR is using URL ref, then requeue after set amount of time
+	// The duration is parsed from an annotation on the OpenAPI CR
+	// We have to requeue like this in case there were updates to the URL source because we can't watch the URL directly
+	if productSynced && openapiCR.Spec.OpenAPIRef.SecretRef == nil {
+		duration := getPollURLInterval(openapiCR.GetAnnotations())
```

Comment:
something small but would it be better to move that func to openapi_types?

Reply:
I removed this function altogether because we decided to remove functionality to let the user specify the OAS URL scrape interval.